app/src/renderer/components/staking/PageBond.vue

```diff
@@ -144,11 +142,7 @@ export default {
       return Math.round(bondedAtoms / this.user.atoms * 100 * 100) / 100 + '%'
     },
     async onSubmit () {
-      if (this.unbondedAtoms === this.user.atoms) {
-        this.$store.commit('notifyError', { title: 'Unbonded Delegates',
-          body: 'You either haven\'t bonded any atoms yet, or some delegates have 0 atoms bonded.' })
-        return
-      } else if (this.unbondedAtoms < 0) {
+      if (this.unbondedAtoms < 0) {
         this.$store.commit('notifyError', { title: 'Too Many Allocated Atoms',
           body: `You've bonded ${this.unbondedAtoms * -1} more atoms than you have.`})
```

Comment:
"You've tried to bond {x} more atoms than you have"

Reply:
:+1: